src/tree.ts

```diff
@@ -22,7 +22,7 @@ export class Tree {
    * @param {boolean} [isBranch] - An option that makes a branch from created tree. Branch can have children.
    */
   public constructor(node: TreeModel, parent: Tree = null, isBranch: boolean = false) {
-    this.buildTreeFromModel(node, parent, isBranch);
+    this.buildTreeFromModel(node, parent, isBranch || Boolean(node.children));
```

Comment:
I want to check if a node has `children` property. If it has `children`, it should be a branch.
It doesn't mater if it has children or no.